### PR TITLE
serializer: fix serialization of 'record' field in experiments for UI

### PR DESCRIPTION
**Pull request by pazembrz**
ref: #1197

### CODE REVIEW

File: backend/tests/integration/records/serializers/json/test_literature.py [hunk 1]
```diff
@@ -673,93 +672,36 @@ def test_literature_detail_serialize_experiment_with_referenced_record(
...
-        "control_number": 1_118_472,
+        "control_number": 1_110_623,
         "institutions": [{"value": "Institute"}],
         "experiment": {"value": "Experiment"},
     }
     create_record_factory("exp", data=experiment_data)
     expected_experiment_data = [
         {
-            "name": "LIGO",
+            "name": "Institute-Accelerator-Experiment",
```
Comment (drjova) [inline backend/tests/integration/records/serializers/json/test_literature.py:682]: we should add a test with linked experiments in literature

Comment (pazembrz) [reply backend/tests/integration/records/serializers/json/test_literature.py:682]: We test this in this exact test, and in every other test I updated.

Comment (drjova) [reply backend/tests/integration/records/serializers/json/test_literature.py:682]: but for literature this is the only one that tests the experiments linked record and it doesn't test the `$ref`

File: backend/inspirehep/records/marshmallow/common/accelerator_experiment.py [hunk 2]
```diff
@@ -37,10 +37,14 @@ def get_control_numbers_to_resolved_experiments_map(self, record):
         return {record["control_number"]: record for record in resolved_records}
 
     def get_resolved_record_or_experiment(self, experiment_records_map, experiment):
-        experiment_record_id = get_recid_from_ref(experiment.get("record"))
+        record_ref = experiment.get("record", None)
```
Comment (drjova) [inline backend/inspirehep/records/marshmallow/common/accelerator_experiment.py:40]:
```suggestion
        record_ref = experiment.get("record")
```

it's `None` by default :)

Comment (pazembrz) [reply backend/inspirehep/records/marshmallow/common/accelerator_experiment.py:40]: I didn't change original code ;)

Comment (drjova) [reply backend/inspirehep/records/marshmallow/common/accelerator_experiment.py:40]: I doesn't matter, could we please change it? :)

Comment (pazembrz) [reply backend/inspirehep/records/marshmallow/common/accelerator_experiment.py]: It's changed already ;)

**Comment by drjova** [comment]
:shipit:

**Pull request merged**